crates/modelardb_common/src/metadata/mod.rs

```diff
@@ -71,6 +71,8 @@ impl MetadataDeltaLake {
 
         let (url_scheme, storage_options) = match object_store_type {
             "s3" => {
+                deltalake::aws::register_handlers(None);
```

Comment:
Fixed. As far as i can tell, it will still be required after 0.20.1 is released. I think at most it will change how we can specify the storage options for MinIO and maybe remove some of the many debug/warn messages that we currently have. To be on the safe side I have created an issue (https://github.com/ModelarData/ModelarDB-RS/issues/221) to evaluate this after the release.

Reply:
@CGodiksen DeltaLake 0.20.1 has no been released to [crates.io](https://crates.io/crates/deltalake/0.20.1). So maybe it is simpler to update to it in this PR and check if it removes the need for `deltalake::aws::register_handlers(None)` or adds additional requirements instead of creating a new PR as a response to #221. Although none of the `deltalake-` crates have been updated from deltalake [0.20.0](https://crates.io/crates/deltalake/0.20.0/dependencies) to [0.20.1](https://crates.io/crates/deltalake/0.20.1/dependencies), so maybe no new code have been included and it simplify fixes an issue in the deltalake crate, e.g., incorrect metadata.

Reply:
@skejserjensen I updated the version but as far as i can tell nothing changed in terms of how we interact with MinIO. `deltalake::aws::register_handlers(None)` is still necessary to avoid the `Error: "Cannot infer storage location from: Unknown scheme: s3"` error. I pushed the update anyway just so we use the newest version and I have added #221 to the issues that get closed by this PR.

Reply:
Sounds good.